x/ibc/04-channel/keeper/handshake.go

```diff
@@ -33,38 +35,45 @@ func (k Keeper) ChanOpenInit(
 	connectionHops []string,
 	portID,
 	channelID string,
+	portCap capability.Capability,
```

Comment:
Passed in by caller as described in spec:

> Note: If the host state machine is utilising object capability authentication (see ICS 005), all functions utilising ports take an additional capability parameter.

Reply:
Yes, that's correct. Working on updating the spec.